enthought/edge-examples
# 1157 edge session ci

Added whoami results in each example

## Testing

- Create a token at edge.enthought.com/hub/token
- Copy the edge_settings.json.example file in each directory to edge_settings.json
- Replace the values in the json file
- All examples should function with `ci dev run`, `ci container run`, `container test`, `preflight run`, `preflight test`
- GitHub actions checks are performed without edge settings
